common/src/main/java/org/jboss/da/common/util/FileUtils.java

```diff
@@ -0,0 +1,25 @@
+package org.jboss.da.common.util;
```

Comment:
We had a discussion also with Vaclav in the office and he also said if it is not essential, it is better to avoid these libraries. We use this library only in a few places. So, I'd like to let it as it is and remove the library later.